kindle.py

```diff
@@ -82,7 +82,7 @@ def set_cookie(self, cookiejar):
 
     def set_cookie_from_browser(self):
         self.set_cookie(browser_cookie3.load())
-
+        
```

Comment:
ident seems wrong

Reply:
Very sorry, I edit this at web browser, let me fix it.